mkdocs.yml

```
@@ -68,33 +68,48 @@ plugins:
             - /docs/*
 nav:
   - 'Overview': index.md
-  - 'Getting Started': getting-started.md
+  - 'Getting Started': 
+    - 'multi-cluster': getting-started.md
+    - 'single-cluster': getting-started.md #needs new doc
```

Should we grow from simpler use case first, to more complex one after?

```suggestion
    - 'Single-Cluster': getting-started.md #needs new doc
    - 'Multi-Cluster': getting-started.md
```